main.nf

```diff
@@ -468,16 +545,18 @@ workflow.onComplete {
 
     // On success try attach the multiqc report
     def mqc_report = null
-    try {
-        if (workflow.success) {
-            mqc_report = ch_multiqc_report.getVal()
-            if (mqc_report.getClass() == ArrayList) {
-                log.warn "[nf-core/epitopeprediction] Found multiple reports from process 'multiqc', will use only one"
-                mqc_report = mqc_report[0]
+    if (!params.show_supported_models) {
```

Comment:
See above, depends on if we just use "show supported models" in a way as "--help". In this case we would not need this here.

Reply:
this does not work, since we need the output of the process `get_software_versions`. And then it would be helpful to also process the pipeline_info etc.